pom.xml

```diff
@@ -510,6 +513,18 @@
                           <ignoreClass>javax.transaction.*</ignoreClass>
                         </ignoreClasses>
                       </dependency>
+                      <dependency>
+                        <!-- The openshift-client duplicates these classes from the kubernetes-client.
```

Comment:
Linked in GUVNOR-3023: https://github.com/fabric8io/kubernetes-client/issues/828